node/src/reactor/main_reactor/tests/builder.rs

```diff
@@ -0,0 +1,164 @@
+#![allow(unused)]
```

Comment:
Can you remove this and set it for the methods that are actually unused?

Reply:
Done